pkg/cli/admin/inspect/pod.go

```diff
@@ -142,6 +155,25 @@ func (o *InspectOptions) gatherContainerRotatedLogFiles(destDir string, pod *cor
 				return
 			}
 
+			if !requestedTime.IsZero() {
+				// make sure rotated logs honor --since and --since-time
+				// when set by the user.
+				if parts := strings.Split(fileName, "."); len(parts) >= 3 {
+					datePart := strings.Split(parts[2], "-")[0]
+					if rotatedTime, err := time.Parse("20060102", datePart); err == nil {
+						if rotatedTime.Before(requestedTime) {
```

Comment:
Last question, we only filter based on dates, not time in `strings.Split(parts[2], "-")[1]`?

Reply:
good point - I didn't realize it before but now that you mention it the part after the `"-"` is the time right? I can change the parse layout so that it takes that into consideration as well.

Reply:
fixed